### ISSUE TITLE
Chrome needs interaction to start audio

### ISSUE THREAD

**Issue by mattgodbolt** [mattgodbolt/jsbeeb#184]
Latest chromes complain about not being able to start audio until an interaction.

```
The AudioContext was not allowed to start. It must be resumed (or created) after a user gesture on the page. https://goo.gl/7K7WLu
```

**Comment by BigEd** [comment]
ah, this maybe explains why I get no beeb until I change model!

**Comment by mattgodbolt** [comment]
Right, or click somewhere in the ui. not sure what to do about this...

**Comment by richtw1** [comment]
I was overriding this Chrome behaviour with a flag, but they seem to have removed it now, and Chrome seems to decide 50/50 whether to allow sound or not now.

You can type:
`soundChip.jsAudioNode.context.resume()`
at the console to get sound back.  I wonder whether it's worth just adding that in keyDown()?

**Comment by mattgodbolt** [comment]
That's a great idea! Thanks. This might also fix #186 . We can also pop a UI element that explains the lack of sound if we notice the node is paused
